Makefile

```diff
@@ -196,3 +196,6 @@ update-codegen-starlark:
 
 update-codegen-ts:
 	toast proto-ts
+
+release-build:
+	toast -f build.toast.yml
```

Comment:
fwiw, i've been wondering if we should move to using Earthly rather than Toast for these kinds of containerized tasks:
https://github.com/earthly/earthly

i've found Toast to be pretty finicky and has a lot of non-intuitive semantics